greatarc/location.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, name, latitude, longitude):
     def get_coordinates(self):
         return f"Latitude: {self.latitude}, Longitude: {self.longitude}"
 
-    def calculate_distance(self, other_location):
+    def calculate_distance(self, other_location, units = "km"):
```

Comment:
Will need a if statement to test whether unit is km or mile, then export different results accordingly.